lib/rpm_package.py

```diff
@@ -140,13 +140,17 @@ def _load(self):
         super(RPM_Package, self)._load()
         try:
             # load distro files
-            files = self.package_data.get('files').get(
-                self.distro.lsb_name).get(self.distro.version)
-
-            self.build_files = files.get('build_files', None)
-            if self.build_files:
-                self.build_files = os.path.join(
-                    self.package_dir, self.build_files)
+            files = self.package_data.get('files', {}).get(
+                self.distro.lsb_name, {}).get(self.distro.version, {})
+
+            default_build_files_dir_rel_path = os.path.join(
+                self.distro.lsb_name, self.distro.version, "SOURCES")
+            build_files_dir_rel_path = files.get('build_files', default_build_files_dir_rel_path)
```

Comment:
SLOF has an empty YAML attribute `build_files`, which means its value is set to `None`.
To avoid breaking things and be backwards compatible, we need to do something like:
`build_files_dir_rel_path = files.get('build_files') or default_build_files_dir_rel_path`
I know, it sucks.